WcaOnRails/db/migrate/20180206211650_add_locations_table.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class AddLocationsTable < ActiveRecord::Migration[5.1]
+  def up
+    create_table :locations do |t|
+      t.integer :user_id, null: false
+      t.integer :latitude
+      t.integer :longitude
+      t.integer :notification_radius
+      t.timestamps
+    end
+
+    add_column :users, :competition_notifications_enabled, :boolean
+  end
+
+  def down
+    drop_table :locations
```

Comment:
Do you really have to define the down separately here? This feels like the kind of simple migration that rails would be able to revers if you defined it in a `def change` instead.